client/dashboard/sites/settings-site-visibility/privacy-form.tsx

```diff
@@ -0,0 +1,144 @@
+import { DataForm } from '@automattic/dataviews';
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	__experimentalToggleGroupControl as ToggleGroupControl,
+	__experimentalToggleGroupControlOption as ToggleGroupControlOption,
+	Button,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { useState } from 'react';
+import type { SiteSettings } from '../../data/types';
+import type { Field } from '@automattic/dataviews';
+import type { UseMutationResult } from '@tanstack/react-query';
+
+interface SitePrivacy {
```

Comment:
Why do we need a new type, can't we just use `SiteSettings` type?

Reply:
This is already addressed below.